### DISCUSSION TITLE
Biometric Types on Android

### DISCUSSION THREAD

**Discussion by NiklasMerz** [NiklasMerz/cordova-plugin-fingerprint-aio#303] [General]
We need help to work with different biometric types on Android

Check why type exactly is available like finger, face etc
Get information wich type has been used for auth request
Only allow certain types for auth request

Anybody with experience on Android and possibly the Biometric API is welcome to comment here and to submit a pull request.

How are you using different types. Please comment.

**Comment by doublerebel** [comment]
Unfortunately, Android does not allow access through the standardized Biometric API to the type of authentication, only the strength of the authentication.  More detail from the API may be added in Android R -- there is an open issue on the Android issue tracker here:
https://issuetracker.google.com/issues/111315641
